swagger.yaml

```diff
@@ -1,22 +1,25 @@
 swagger: '2.0'
 info:
   title: Analytics Core API
-  version: '0.1'
+  version: '0.2'
 basePath: /api/v1/
+host: "f8a-analytics-2445582058137.production.gw.apicast.io"
 consumes:
   - application/json
 produces:
   - application/json
+
+securityDefinitions:
```

Comment:
I guess @yzainee is the right person to add doc related to Auth tokens.

Reply:
This Again added to realise Interactive Mode. I have Tested it. Working fine